Api/Client.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+    namespace Metrilo\Analytics\Api;
+
+    use \Metrilo\Analytics\Api\Validator;
+    use \Metrilo\Analytics\Api\Connector;
+
+    class Client {
+    
+//        private $url  = 'https://postb.in/kzTiQWNl';
+        private $url  = 'https://postman-echo.com/post';
+    
+        public function __construct(
+            $time,
+            $token,
+            $platform,
+            $pluginVersion
+        ) {
+            $this->backendParams['time']          = $time;
+            $this->backendParams['token']         = $token;
+            $this->backendParams['platform']      = $platform;
+            $this->backendParams['pluginVersion'] = $pluginVersion;
+        }
+    
+        public function backendCall($path, $body) {
+            $connection      = new Connection();
```

Comment:
What's with all the spaces?

Reply:
Alignment for deleted variable. Removed.